upup/pkg/fi/cloudup/awsup/aws_utils.go

```diff
@@ -197,3 +200,28 @@ func EC2TagSpecification(resourceType string, tags map[string]string) []*ec2.Tag
 
 	return []*ec2.TagSpecification{specification}
 }
+
+// GetResourceName32 will attempt to calculate a meaningful name for a resource given a prefix
+// Will never return a string longer than 32 chars
+func GetResourceName32(cluster string, prefix string) string {
```

Comment:
this isn't necessarily specific to AWS so i'm wondering if it would better belong in `pkg/util` or `util/pkg` (why do we have both??)

Reply:
For now the 32 chars name is limited to AWS and the other packages don't seem to be suitable for this. I think it's good enough for now.